### PR TITLE
add default mix rule

### PR DESCRIPTION
**Pull request by innovimax-jim**

### CODE REVIEW

**Comment by innovimax-jim** [comment]
have to fixe a bug

**Comment by innovimax-jim** [comment]
need to implement some exceptions

**Comment by innovimax-jim** [comment]
add missing file exception

File: src/main/java/innovimax/mixthem/MixThem.java [hunk 1]
```diff
@@ -30,10 +33,32 @@ private static void run(String[] args) {
         }
     }
 
-    private static void processFiles(String files1, String file2) throws MixException { 
-        //TODO
+    private static void processFiles(String file1, String file2) throws MixException {
+        try {
```
Comment (innovimax) [inline src/main/java/innovimax/mixthem/MixThem.java]: you should add a param rule to the method

File: src/main/java/innovimax/mixthem/MixThem.java [hunk 2]
```diff
@@ -30,10 +33,32 @@ private static void run(String[] args) {
...
+        }
+
     }   
 
+    private static void processRule1(String file1, String file2) throws MixException, IOException { 
+        File file = new File(file1);
+        FileReader in = new FileReader(file);
+        FileWriter out = new FileWriter(file.getName() + "mix");
+        int c;
+        while ((c = in.read()) != -1) {
```
Comment (innovimax) [inline src/main/java/innovimax/mixthem/MixThem.java]: might be better to use a byte array

File: src/main/java/innovimax/mixthem/MixThem.java (hunk 1 above)
Comment (innovimax-jim) [inline src/main/java/innovimax/mixthem/MixThem.java]: ok I will do it

File: src/main/java/innovimax/mixthem/MixThem.java (hunk 2 above)
Comment (innovimax-jim) [inline src/main/java/innovimax/mixthem/MixThem.java]: maybe you're right

File: src/main/java/innovimax/mixthem/MixThem.java (hunk 1 above)
Comment (innovimax-jim) [inline src/main/java/innovimax/mixthem/MixThem.java]: but first I need to check rule param in arguments in another issue
so not here

File: src/main/java/innovimax/mixthem/MixThem.java (hunk 2 above)
Comment (innovimax-jim) [inline src/main/java/innovimax/mixthem/MixThem.java]: I do that now

**Comment by innovimax** [comment]
implements part of #18 and #5

**Pull request merged**